### DISCUSSION TITLE
How to use Beaker on two computers?

### DISCUSSION THREAD

**Discussion by programmarchy** [beakerbrowser/beaker#1652] [General] [answered]
Hi, I installed Beaker on my Mac, but also want to use it on my desktop.

I know that "device sync" is a planned feature, but is there a manual process that one could follow in the meantime? Maybe fork my system drive, or edit a configuration file to map a drive address to the system drive?

Don't need a step-by-step, just a pointer in the right direction...

Any tips would be appreciated.

**Comment by brechtcs** [comment]
Just to be clear, you're asking for a way to migrate your setup from one device to another, rather than using two devices at the same time, right?

@pfrazee Maybe this is something that should be implemented in `hyperdrive-daemon`, to prevent the footguns of doing this manually.

**Comment by danrobi11** [comment]
I have a question a bit related
With I2P and IPFS its recommended to add multiple router running in same network(modem) as 'familly router' Like that theres only one router ID so it doesnt confuse the I2P/IPFS network.
Can we run multiple beaker daemon with different device on same network, or is this a problem like with I2P/IPFS?

**Comment by pfrazee** [comment, marked as answer]
If you're looking to _move_ between devices, use the following steps _precisely_: https://twitter.com/BeakerBrowser/status/1263610518822100992?s=20

If you're looking to work on your projects from multiple computers, no that's really what we need device sync for. It's just not possible to write to a Hyperdrive from more than one machine right now.

**Comment by programmarchy** [comment]
@pfrazee Sounds good, thanks for the link!

@brechtcs I think what I was getting after was sharing the same identity on two computers. The identity is just a cert and the root or "system" Hyperdrive address, right?

Something like...

Computer 1:
cert: Identity X
url: System Drive A

Computer 2:
cert: Identity X
url: Fork of System Drive A or System Drive B

Sounds like I should just have a separate identity for each computer until Device Sync is available.

**Reply by danrobi11** [reply]
'Sounds like I should just have a separate identity for each computer until Device Sync is available'
Many 'separate identity' over same IP could be tricky. Im waiting for an official answer about this.
